taskfiles/k8s.yml

```diff
@@ -70,25 +70,26 @@ tasks:
       - |
         go test -v ./... -coverprofile cover.out -timeout 20m
 
-  build-operator-images:
+  build-container-image:
```

Comment:
Are these changes related to the nightly helm release or the removal of the vectorized repository? If it's the nightly helm release, could you explain why this change is needed? It looks to primarily be a refactor to me?